internal/exec/util/passwd.go

```diff
@@ -234,13 +234,13 @@ func (u Util) CreateGroup(g types.PasswdGroup) error {
 			strconv.FormatUint(uint64(*g.Gid), 10))
 	}
 
-	if g.PasswordHash != "" {
-		args = append(args, "--password", g.PasswordHash)
+	if g.PasswordHash != nil && *g.PasswordHash != "" {
```

Comment:
The `foo != nil && *foo != ""` pattern is pretty common now. Might be worth some shorthands in `utils` that encodes the `nil -> ""` to make it less verbose. E.g. something like

```go
if utils.PtrToStr(g.PasswordHash) != "" {
    ...
}
```

And similarly for the other types. Anyway, can do this later of course.

Reply:
:+1: to doing it later, I agree it's quite annoying.